sdo/dataset_distribution_sdo_context.jsonld

```diff
@@ -27,8 +27,36 @@
       "@type": "sdo:EntryPoint"
     },
     "size": {
-        "@id": "sdo:contentSize",
-        "@type": "sdo:Text"
+      "@id": "sdo:contentSize",
+      "@type": "sdo:Text"
+    },
+    "dates": {
+      "@id": "sdo:temporalCoverage",
+      "@type": "sdo:DataCatalog"
```

Comment:
Should be one of these data types DateTime, Text or URL. The `Text ` probably is the best option.